consensus/bor/bor_test.go

```diff
@@ -142,48 +140,3 @@ func TestEncodeSigHeaderJaipur(t *testing.T) {
 	hash = SealHash(h, &params.BorConfig{JaipurBlock: 10})
 	require.Equal(t, hash, hashWithoutBaseFee)
 }
-
-// TestCheckpoint can be used for to fetch checkpoint
-// count and checkpoint for debugging purpose.
-// Also, this is kept only for local use.
-func TestCheckpoint(t *testing.T) {
```

Comment:
Why do you removed this test?

Reply:
It was only for debugging purpose (checking if all entities are correct, marshalling/unmarshalling takes place correctly or not, etc). Now that we have the mock server, all these checks can be easily added in the same test and can also be used for debugging by changing the heimdall client url.